packages/components/src/components/error/ErrorBoundary.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ErrorInfo } from 'react';
+import { getServerContext } from '@labkey/api';
+
+import { ErrorPage } from './ErrorPage';
+
+interface ErrorBoundaryState {
+    error: Error
+    errorInfo: ErrorInfo
+    hasError: boolean
```

Comment:
Since we initialize this.state.error as undefined you should be able to compute `hasError` by checking `this.state.error !== undefined` correct?

Reply:
I've updated this with edc3f40.